src/nvim/ex_getln.c

```diff
@@ -349,6 +349,43 @@ static void get_healthcheck_cb(char_u *path, void *cookie)
   }
 }
 
+// Return an user command when it can do incsearch highlighting, NULL otherwise.
+// Will also advance the pointer to the end of the command name, if appropriate.
+static ucmd_T *usrcmd_can_incsearch(char_u *cmd_name, char_u **p)
```

Comment:
Minor nit but for the sake of consistency:
```suggestion
static ucmd_T *uc_can_incsearch(char *cmd_name, char **p)
```

We also try to avoid adding new usages of `char_u`, so for new functions just use `char` and cast where necessary.

Reply:
Ok